LINUX/virtio_netmap.h

```diff
@@ -677,6 +685,20 @@ virtio_netmap_intr(struct netmap_adapter *na, int onoff)
 	}
 }
 
```

Comment:
Defining this function is unnecessary, because nothing is going to change the values listed in `struct nm_config_info`...
Here you are not updating them indeed.